src/components/FeaturePanel/PublicTransport/PublicTransport.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import { useRouter } from 'next/router';
+import { requestLines } from './requestRoutes';
+import { PublicTransportWrapper } from './PublicTransportWrapper';
+import { FeatureTags } from '../../../services/types';
+import { LineNumber } from './LineNumber';
+
+interface PublicTransportProps {
+  tags: FeatureTags;
+}
+
+export const PublicTransport: React.FC<PublicTransportProps> = ({ tags }) => {
+  const isPublicTransport =
+    Object.keys(tags).includes('public_transport') ||
+    tags.railway === 'station';
+
+  if (!isPublicTransport) {
+    return null;
+  }
+
+  const [data, setData] = useState(null);
```

Comment:
Could you please split this component after the condition? React forbids using hooks after an early return.

Just make another component `PublicTransportInner` and put there everything from line 21 and so on.